spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/channel/SqlChannelMessageStoreQueryProvider.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
```

Comment:
This is a new class, so only a current year.

Reply:
Yep. Thanks.